README.md

```diff
@@ -45,7 +48,11 @@ This repo requires the following to be installed on your machine:
 
 ## Tasks
 
-Each task should be completed in the `main.py` file. The tasks are as follows:
+Each task should be completed in the `main.py` file. During the interview, you will be asked to walk through
+your solution and explain your thought process. You will also be allowed to use the internet to look up
+documentation and other resources. If you have any questions, please ask.
```

Comment:
Maybe something here beyond just offering to answer questions, but clarifying that you are also here to talk through their approach/thinking and help arrive at an answer, as long as they are driving. We want them to figure it out but not get hung up on something like syntax or whatever. 

We obviously can say this verbally, but it's helpful for someone else besides you doing the interview as well.